### DISCUSSION TITLE
Attempt to move tests by default with MoveCommand

### DISCUSSION THREAD

**Discussion by Naoray** [livewire/livewire#4820] [3. Feature ideas]
Atm when executing the `php artisan livewire:move` command you have to specify the `--test` flag if you also want to move the test. IMO if there is a test, you probably always want to rename/move the test as well.

Could we just drop the `--test` flag and always attempt to rename the test?

It's only a small change, but I always forget to append the flag when renaming components :man_shrugging:

**Comment by zepfietje** [comment]
I think it makes sense to move the test as well when it's present. Feel free to create a PR!

**Comment by joshhanley** [comment]
@Naoray yep, that gets me too. PR's are welcome! 🙂
